### PR TITLE
Write changelog messages & remove component reference

### PR DESCRIPTION
**Pull request by tuommaki**
Component reference in version bundle caused misunderstandings among SEs
and customers. SIG Operator decided to remove it. To make this
effective, prepare for new release by writing appropriate changelog
messages.

### CODE REVIEW

File: service/controller/aws/v2/version_bundle.go [hunk 1]
```diff
@@ -10,17 +10,17 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "Cluster Operator",
-				Description: "TODO",
+				Component:   "cluster-operator",
```
Comment (tuommaki) [inline service/controller/aws/v2/version_bundle.go]: I changed this due to https://github.com/giantswarm/fmt/blob/master/releases/how_write_changelog_message.md#rules and common style how we write operator names.

**Comment by tuommaki** [comment]
> This will make the operator panic when run. See https://github.com/giantswarm/versionbundle/blob/b4b856d2de9ec705a7b55163fa7849048b03d9fe/bundle.go#L174-L176.

Very true. Are there any reason why it shouldn't be removed from `versionbundle`?

**Comment by xh3b4sd** [comment]
AFAIK it was just a wild guess that version bundles should have components. When there is a reason to drop the validation lets do it.

**Comment by rossf7** [comment]
@tuommaki I think there should be an entry because the chart resource is enabled.

Could you change the message to "Install chart-operator in kube-system namespace." As this is what the chart resource is doing.

**Comment by tuommaki** [comment]
@xh3b4sd PTAL :heart:

File: service/controller/aws/v2/version_bundle.go [hunk 2]
```diff
@@ -10,17 +10,17 @@ func VersionBundle() versionbundle.Bundle {
...
+				Description: "Install chart-operator in kube-system namespace.",
```
Comment (rossf7) [inline service/controller/aws/v2/version_bundle.go]: <3

Comment (xh3b4sd) [reply service/controller/aws/v2/version_bundle.go]: That should be `Installed` right? Isn't it always past tense and the verb is the very first thing?

File: service/controller/aws/v2/version_bundle.go [hunk 3]
```diff
@@ -10,17 +10,17 @@ func VersionBundle() versionbundle.Bundle {
...
 				Kind:        versionbundle.KindChanged,
 			},
-		},
-		Components: []versionbundle.Component{
 			{
-				Name:    "aws-operator",
-				Version: "1.0.0",
+				Component:   "cluster-operator",
+				Description: "Misleading component reference to aws-operator removed.",
```
Comment (xh3b4sd) [inline service/controller/aws/v2/version_bundle.go]:
```
Removed misleading component reference to aws-operator.
```

Comment (xh3b4sd) [reply service/controller/aws/v2/version_bundle.go]: Same in `v1`.

Comment (tuommaki) [reply service/controller/aws/v2/version_bundle.go]: You are right. It should be past tense + verb first. Do you suggest `v1` should be also changed although it's already released?

File: service/controller/aws/v2/version_bundle.go (hunk 2 above)
Comment (tuommaki) [reply service/controller/aws/v2/version_bundle.go]: Yes it should, I'll fix.

File: service/controller/aws/v2/version_bundle.go (hunk 3 above)
Comment (xh3b4sd) [reply service/controller/aws/v2/version_bundle.go]: Should be ok to fix it there as well. The releases are generated anyway so the fix should be applied immediately.

**Pull request merged**